pkg/validation/validate.go

```diff
@@ -188,9 +189,50 @@ func (v *Validator) Validate() error {
 		v.Log.Logger.Info().Msg("No SSH host keys found bundled in the system")
 	}
 
+	// Check RHEL family specific service validations
+	if err := v.ValidateRHELServices(); err != nil {
```

Comment:
I woudl make this more generic than RHEL in order to expand the services to check int he future for other systems. For example for Fedora we need to validate if the getty service is unmasked

Reply:
Wrapped into a generic method which also validates the getty service